### PR TITLE
feat: add multi select to quick search

### PR DESCRIPTION
**Pull request by Christof-P**
Attempt to help with feat #1737 

Any feedback is welcome

### CODE REVIEW

File: apps/studio/src/components/quicksearch/QuickSearch.vue [hunk 1]
```diff
@@ -146,6 +146,7 @@ export default Vue.extend({
       // /me *evil laugh*
       result['ctrl+p'] = this.selectUp
       result['ctrl+n'] = this.selectDown
+      result['right'] = this.persistentSearchEnter
```
Comment (rathboma) [inline apps/studio/src/components/quicksearch/QuickSearch.vue:149]: how do we signal to users that pressing right will keep the quick search open?

**Review by rathboma — COMMENTED** [review]
This is great. Happy to merge it. Quick question -- how can we advertise this feature?

File: apps/studio/src/components/quicksearch/QuickSearch.vue (hunk 1 above)
Comment (Christof-P) [reply apps/studio/src/components/quicksearch/QuickSearch.vue:149]: Good question, I hadn't really thought about it. VS code doesn't really show this well, so I couldn't really get inspiration from there. 
We could add it to the initial quick search popup (I'll add a picture below of what that could look like), but I don't know if that is the right place to do this.
Screenshot from 2023-11-30 09-49-28

**Comment by rathboma** [comment]
I checked VSCode, it looks like it does your first suggestion - detect if at the end of the input, and only open then.

I like that idea, makes sense. I'd like to stay consistent with VSCode on this

**Comment by Christof-P** [comment]
Found a way to resolve the bug.
It works now like in VSCode.

File: apps/studio/src/components/quicksearch/QuickSearch.vue (hunk 1 above)
Comment (rathboma) [reply apps/studio/src/components/quicksearch/QuickSearch.vue:149]: You could have an entry for each:
- Open in background: [Right Arrow]
- Alt in background: [Meta + Right Arrow]

**Comment by Christof-P** [comment]
I added the labels as requested, I'll add a screenshot of them below.
Thank you for letting me work on this!

search change

**Comment by rathboma** [comment]
So good. Thanks so much!

**Review by rathboma — APPROVED** [review]
:rocket: :rocket:

**Pull request merged**